test/agentchat/test_conversable_agent.py

```diff
@@ -25,6 +25,8 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
+gpt4_config_list = [{"model": "gpt-4"}, {"model": "gpt-4-turbo"}, {"model": "gpt-4-32k"}, {"model": "gpt-4o"}]
```

Comment:
Hey @qingyun-wu, I am a little confused when we use "model" vs. "tags" as key. Can you help me understand the difference please?

Reply:
A tag corresponds to a family of models with certain shared properties. E.g., "gpt-3.5-turbo", "gpt-3.5-turbo-0613" may have the name tag "gpt-3.5-turbo".